app/controllers/schools/manual_readings_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Schools
+  class ManualReadingsController < ApplicationController
+    load_and_authorize_resource :school
+    load_and_authorize_resource through: :school
+
+    before_action :set_breadcrumbs
+
+    def show
+      @meter_start_dates = %i[electricity gas].select { |fuel_type| @school.configuration.fuel_type?(fuel_type) }
+                                              .index_with do |fuel_type|
+        @school.configuration.meter_start_date(fuel_type)
+      end
+      return if @meter_start_dates.empty?
+
+      build_manual_readings
+      @readings = @school.manual_readings.sort_by(&:month)
+    end
+
+    def update
+      @school.update(resource_params)
+      if @school.save
+        redirect_to school_manual_readings_path(@school), notice: t('common.saved')
+      else
+        show
+        render :show
+      end
+    end
+
+    private
+
+    def build_manual_readings
+      end_date = @meter_start_dates.values.compact.max || Date.current
+      return if end_date < 1.year.ago
```

Comment:
Please could we have some documentation of the logic here and what the code is trying to achieve?
i.e. Are we trying to ensure we have 13 months of readings, if we don't have the actual readings, fill in with manual readings. 
Looks like we are finding the latest start time for existing meter data across fuel types. If this is over a year ago, then return as presume this means we have enough data? Is this true though if we're missing data at the end? Should it be 13 months?